### ISSUE TITLE
visibilité et notification aussi au niveau du RDV

### ISSUE THREAD

**Issue by yaf** [betagouv/rdv-solidarites.fr#1061]
Choix de la "Visibilité usager" : pouvoir surcharger le comportement du motif dans un RDV particulier

**Comment by adipasquale** [comment]
toujours plus de complexité 🤯 
je pense que dans un premier temps ça serait bien d'expliciter ce qu'il va se passer en termes de notifs à la création d'un RDV. L'idée serait d'afficher une page qui explique quelles notifs (SMS/Mail) vont être envoyées à quels usagers, et si possible pourquoi : combinaison de la visibilité du motif + des préférences usager (+ autre chose que j'oublie ? ) 

et éventuellement après ça on pourra rajouter une case "override et n'envoyer aucune notif".

qu'en penses-tu @yaf ?

**Comment by yaf** [comment]
Oui, je pense même que nous pourrions ici combiner quelque chose lié à la structure familiale... 

Je ferme celui-ci. Nous reprendrons le sujet sur un autre ticket.
